dev-java/browserlauncher2/browserlauncher2-1.3-r2.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source"
+
+inherit java-pkg-2 java-ant-2
+
+MY_PV="$(ver_rs 1- _)"
+
+DESCRIPTION="A library that facilitates opening a browser from a Java application"
+HOMEPAGE="http://browserlaunch2.sourceforge.net/"
+SRC_URI="https://downloads.sourceforge.net/project/browserlaunch2/browserlauncher2/${PV}/BrowserLauncher2-all-${MY_PV}.jar"
+
+LICENSE="LGPL-2.1"
+SLOT="1.0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+
+DEPEND=">=virtual/jdk-1.8:*"
+RDEPEND=">=virtual/jre-1.8:*"
+BDEPEND="app-arch/unzip"
+
+S="${WORKDIR}"
+
+src_prepare() {
+	default
+	unpack ${A}
+	find . -name "*.class" -exec rm -v {} \; || die
+	# fixing build.xml
+	sed -i -e "s: includes=\"\*\*/\*\.class\"::g" "${S}/build.xml" || die
+
+	iconv -f ISO-8859-1 -t UTF8 -o /var/tmp/portage/dev-java/browserlauncher2-1.3-r2/work/source/at/jta/Regor.java~ \
+		/var/tmp/portage/dev-java/browserlauncher2-1.3-r2/work/source/at/jta/Regor.java || die "recoding failed"
+	mv -f /var/tmp/portage/dev-java/browserlauncher2-1.3-r2/work/source/at/jta/Regor.java{~,} || die "cannot rename"
+}
```

Comment:
cool! just should use `${S}` instead of hardcoded paths :-)

Reply:
Of course. Could you  change it on merge, please?